cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/compute/GcpInstanceResourceBuilder.java

```diff
@@ -124,6 +128,14 @@
                 projectId, location.getAvailabilityZone().value(), template.getFlavor()));
         instance.setDescription(description());
         instance.setName(buildableResource.get(0).getName());
+        Optional<CloudFileSystemView> cloudFileSystemView = group.getIdentity();
```

Comment:
you need the CloudIdentityType = LOG right ?

Reply:
we should set idbroker one as well 

(also we can set both service-accounts for gcp in that case, but probably should not be different against other cloud providers)